processor/transformprocessor/internal/traces/expression_test.go

```diff
@@ -0,0 +1,101 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package traces
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/collector/model/pdata"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/transformprocessor/internal/common"
+)
+
+func hello() exprFunc {
+	return func(span pdata.Span, il pdata.InstrumentationLibrary, resource pdata.Resource) interface{} {
+		return "world"
+	}
+}
+
+func Test_newGetter(t *testing.T) {
+	registerFunction("hello", hello)
+	defer unregisterFunction("hello")
```

Comment:
would be good to avoid this in tests and in general in the project. We can change "newGetter" to accept a `type AvailableFunc map[string]interface{}`